test_data/asm/book/declarations.asm

```diff
@@ -5,9 +5,8 @@ mod utils {
         0 => 0,
         _ => arr[len - 1] + sum(len - 1, arr)
     };
-    // A simple function that returns the input incremented by one,
-    // as an expression.
-    let incremented: expr -> expr = |x| x + 1;
+    // A simple function that returns the same input as an expression.
```

Comment:
are these book changes related?

Reply:
minor, but why not keep the `incr` instruction and call it from `main`? it is a bit more "interesting" of an example